.ci/update-github-milestone.sh

```diff
@@ -12,10 +12,11 @@ MILESTONE_ID=$(curl -s \
                 -X GET https://api.github.com/repos/checkstyle/checkstyle/milestones?state=open \
                 | jq ".[0] | .number")
 
-echo VERSION="$VERSION"
+LATEST_VERSION="${VERSION%-*}"
+echo LATEST_VERSION="$LATEST_VERSION"
```

Comment:
Why this required?

Reply:
To extract. For eg: 

If `10.4-SNAPSHOT` then it will extract `10.4`

Reply:
If user set 10.4-SNAPSHOT, your update in pom will be disaster.
Lets have user be responsible to provide version without snapshot.

Reply:
I think you are ok to revert changes in this file

Reply:
Please put description to input parameter
https://github.blog/changelog/2020-07-06-github-actions-manual-triggers-with-workflow_dispatch/ to explain what value is expected.

Please share screenshot of description

Reply:
We already have a description in input parameter:
https://github.com/checkstyle/checkstyle/blob/master/.github/workflows/bump-version-and-update-milestone.yml#L10

Screenshot:
![184149436-19f98d86-0bf5-4168-a334-556604b4c320](https://user-images.githubusercontent.com/71710042/184819328-1bed7abb-0718-4b3b-af80-73677f8f6e00.png)

Reply:
Put in it ` without SNAPSHOT`

Reply:
@romani you mean `New Version without SNAPSHOT` or just `without SNAPSHOT`

Reply:
New Version (without '-SNAPSHOT')

Reply:
Changes Done